guide/advice-for-new-collaborators.rst

```diff
@@ -312,43 +314,8 @@ of the state variables then doing it on demand is more efficient.
 The options here are namelist selectable at runtime and the impact
 on total runtime can be easily measured and compared.
 
-Adding a DART interface to a new model
---------------------------------------
-
-DART provides a script ``new_model.sh`` which will create the necessary files
-for a new model interface.
-Enter ``./new_model.sh``, then the desired model name and location module separated
-by spaces. This will create the necessary files to get started.
-
-For example to create a model interface for a model called BOUMME which uses
-the 3D sphere location module:
-
-.. code-block::  text
-
-   cd models
-   ./new_model.sh BOUMME threed_sphere
-
-This will create an BOUMME model directory with the following files:
-
-.. code-block:: text
-
-     BOUMME/
-     ├── model_mod.f90
-     ├── readme.rst
-     └── work
-         ├── input.nml
-         └── quickbuild.sh
-
-- ``model_mod.f90`` is where to add the :doc:`required model_mod routines<required-model-mod-routines>`.
-- ``readme.rst``  is a stub to add documenation for your model interface.  
-- ``quickbuild.sh`` is used to compile DART for your model.
-
-
-Templates are chosen based on location module input. The currently supported
-location templates are for 3D and 1D modules, with the possibility for more
-in the future. At the moment, ``threed_sphere``, ``threed_cartesian``, and
-``oned``  will produce model_mod.f90 code that compile will sucessfully with ``./quickbuild.sh``.
+Where can I learn more about the process of creating a DART interface to a new model?
+-----------------------------------------------------------------------------------
```

Comment:
I would remove this section, and just add 

instructions-for-porting-a-new-model-to-dart.rst to the table of contents for the 'RUN DART WITH YOUR MODEL' section.